### PR TITLE
CVE-2015-1281 and CVE-2016-5152

### PR DESCRIPTION
**Pull request by David-Egan**
In Krutz's section

### CODE REVIEW

File: cves/CVE-2016-5152.yml [hunk 1]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  A function inside of OpenJPEG which is used by PDFium in Google Chrome
+  didn't prevent integer overflow giving attackers the ability to cause 
+  denial of service with heap-based buffer overflow attacks (explain dos and hbbo).
```
Comment (xdmds) [inline cves/CVE-2016-5152.yml]: is the "(explain dos and hbbo)" a note to yourself?

File: cves/CVE-2015-1281.yml [hunk 2]
```diff
@@ -29,20 +29,22 @@ reviews:
...
+repo: https://chromium.googlesource.com/v8/v8/
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: dff368031150a1033a1a3c913f8857679a0279be
   :note: ''
 - :commit: 9fb1e9fc1f3829f9597f830e599fc40fadc84db3
-  :note: ''
+  :note: 'Reimplement fix this time it passed tests.  Last commit for this fix.'
```
Comment (xdmds) [inline cves/CVE-2015-1281.yml]: see cves/CVE-2011-3092.yml and match that formatting

File: cves/CVE-2015-1281.yml [hunk 3]
```diff
@@ -29,20 +29,22 @@ reviews:
...
 - :commit: 7faa3067e4b664eb19a7a682697f5b387a130728
   :note: ''
 - :commit: 0f291b9f331a04c55e0d050bfaef20b1c6a6af1c
-  :note: ''
-vccs: []
+  :note: 'Revert attempt at fix as it was failing tests.'
+vccs: 
+- :commit: 52aa3b1bcf3079fc72a2b24a73dff78017eb5e7a
+  :note: 'Bug fix for issue 200923002'
```
Comment (kalebdavis) [inline cves/CVE-2015-1281.yml]: It'd be nice to see more information about this VCC. Go into the issue a bit and explain how it introduced a new vulnerability.

File: cves/CVE-2015-1281.yml [hunk 4]
```diff
@@ -82,9 +84,9 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
   answer:
```
Comment (kalebdavis) [inline cves/CVE-2015-1281.yml]: You found a date here, so you must have found some evidence as to how the vulnerability was found. Can you elaborate in the 'answer' block?

File: cves/CVE-2016-5152.yml [hunk 5]
```diff
@@ -173,4 +178,4 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: I believe design mistakes were made by the Chromium team.  By incorporating the open source OpenJPEG2000 project Chromium was exposed to vulnerabilites in that project.  Security doesn't seem to have been as big a concern to the developers of OpenJPEG as compared to Chromium.    It is likely that many other vulnerabilities have been found/exist as a result of the use of third-party software like OpenJPEG2000.
```
Comment (kalebdavis) [inline cves/CVE-2016-5152.yml]: sp: vulnerabilities

**Comment by kalebdavis** [comment]
CVE-2015-1281: 4 upvotes
CVE-2016-5152: 2 upvotes

File: cves/CVE-2015-1281.yml [hunk 6]
```diff
@@ -93,8 +95,8 @@ subsystem:
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer: 
-  name: 
+  answer:
+  name: Platform->Extensions
```
Comment (btarr) [inline cves/CVE-2015-1281.yml]: since the repo is the V8 repo wouldn't the subsystem be V8?

File: cves/CVE-2016-5152.yml (hunk 1 above)
Comment (David-Egan) [inline cves/CVE-2016-5152.yml]: Yes, I am trying to balance giving the reader the details without overwhelming them with technical information and jargon.  I will edit this and explain to the potential impact of dos and hbbo to try to achieve this balance.

File: cves/CVE-2015-1281.yml (hunk 2 above)
Comment (David-Egan) [inline cves/CVE-2015-1281.yml]: Thanks! Adjustments made in 1cee0c6

File: cves/CVE-2016-5152.yml (hunk 1 above)
Comment (David-Egan) [inline cves/CVE-2016-5152.yml]: Fixed in 2adc856

File: cves/CVE-2015-1281.yml (hunk 6 above)
Comment (David-Egan) [inline cves/CVE-2015-1281.yml]: True, extensions cause this vulnerability but it occurs within the V8 subsystem.  I will correct that.

File: cves/CVE-2016-5152.yml (hunk 5 above)
Comment (David-Egan) [inline cves/CVE-2016-5152.yml]: Corrected in bab13e11

File: cves/CVE-2015-1281.yml (hunk 6 above)
Comment (David-Egan) [inline cves/CVE-2015-1281.yml]: After some more research it looks like the Blink>SVG subsystem is even more specific as it controls the rendering of images such as the offending images for this vulnerability.

File: cves/CVE-2015-1281.yml (hunk 4 above)
Comment (David-Egan) [inline cves/CVE-2015-1281.yml]: The bug report was posted with info on how to reproduce the bug but I couldn't find anything more specific anywhere.  I did update this field to show which user reported the bug in ef664bd .

File: cves/CVE-2015-1281.yml (hunk 3 above)
Comment (David-Egan) [inline cves/CVE-2015-1281.yml]: Added some more info in efa4cf4

**Pull request merged**